src/sentry/utils/assets.py

```diff
@@ -1,5 +1,21 @@
+from __future__ import annotations
+
+import os.path
+
+from cachetools.func import ttl_cache
 from django.conf import settings
 
+from sentry.utils import json
+
+
+@ttl_cache(ttl=60)
+def _frontend_versions() -> dict[str, str]:
+    try:
+        with open(os.path.join(settings.CONF_DIR, "settings", "frontend-versions.json")) as f:
```

Comment:
What is generating these?

I would expect this to change for every deploy of the frontend application, as part of the frontend deploy are we pushing these files out to all of the webs somehow?

Reply:
configmaps, yes